LDAR_Sim/src/programs/method.py

```diff
@@ -439,27 +439,27 @@ def check_weather(self, weather, curr_date, site: Site) -> bool:
         PREICIP = "precip"
 
         # TODO change to getters
-        lat = site._lat
-        long = site._long
+        lat = np.where(weather.latitude == math.floor(site._lat))[0][0]
```

Comment:
Can you explain what we are getting back if it is not the same latitude/longitude - Why is this lookup necessary?

Reply:
The reason this lookup happens is because the weather object that we build requires us to key into it to find the temp.
Instead what we'll do is save this key as a sites property instead of doing this lookup every time.